portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,108 @@
-    <title>My Portfolio</title>
+    <title>Matthew Wong 🎯</title>
+    <link rel="shortcut icon" type="image/jpg" href="/images/Raptors.png"/>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
+  <body class="cursor">
+    <!--Top Half of Site-->
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <!--Introduction-->
+      <h1 class="headerFont">Matthew Wong</h1>
+      <img src="/images/ProfilePic.png" width="180" height="180" class="profilePic"/>
+      <p class="mediumFont">Swift iOS Developer • Startups 🚀</p>
+      <p class="fonts">Hello, nice to meet you! I'm a mobile developer from Toronto, Canada. I'm currently working @ theScore as an iOS Developer, and previously worked @ Shopify in a technical support/ecommerce consulting role!</p>
+
+      <!--Social Media Icons-->
+      <a href="https://github.com/MattWong-ca"><img src="/images/GitHub.png" class="socialMediaIcons"/></a>
+      <a href="https://www.linkedin.com/in/mattwong-ca/"><img src="/images/LinkedIn.png" class="socialMediaIcons"/></a>
+      <a href="https://twitter.com/mattwong_ca"><img src="/images/Twitter.png" class="socialMediaIcons"/></a>
+      <a href="https://www.instagram.com/mattwong.ca/"><img src="/images/Instagram.png" class="socialMediaIcons"/></a>
+      <a href="https://www.f6s.com/mattwong.ca"><img src="/images/F6S.png" class="socialMediaIcons"/></a>
+
     </div>
+
+    <!--Experience-->
+    <div class="test">
+        <p class="headerFont"><a style="text-decoration:none;">Experience 💼</a></p>
+        <a href="https://www.thescore.com/"><img src="/images/theScore.png" class="experienceLogos"/></a>
+        <a href="https://www.shopify.com/"><img src="/images/Shopify.jpeg" class="experienceLogos"/></a>
+        <a href="https://www.foundersboost.com/"><img src="/images/FoundersBoost.jpeg" class="experienceLogos"/></a>
+    </div>
+    </br>
+
+    <!--Skills-->
+    <div class="test">
+        <p class="headerFont">Skills 🛠</p>
+        <p class="fonts">Languages/Frameworks: Swift, Java, C#, C++, JavaScript, HTML, CSS</p>
+        <p class="fonts">Tools/Technologies: Git/GitHub</p>
+        <p class="fonts">APIs: </p>
+    </div>
+    </br>
+
+    <!--Projects-->
+    <div class="test">
+        <p class="headerFont">Projects 💡</p>
+        <p class="fonts">[PLACEHOLDER: WILL USE CARDS OR CAROUSEL TO DISPLAY PROJECTS]</p>
+    </div>
+    </br>
+
+    <!--Interests-->
+    <div class="test">
+        <p class="headerFont">Interests 🏀</p>
+        <a href="#"><img src="/images/Raptors.png" class="experienceLogos"/></a>
+        <a href="#"><img src="/images/Travelling.png" class="experienceLogos"/></a>
```

Comment:
Remove <a href> wrapper or add real link.

These images are clickable but because you're referencing "#" don't actually open a link.